src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/AnalyzePackageStatus.cs

```diff
@@ -17,7 +17,11 @@ public class AnalyzePackageStatus : IAnalyzeResultProvider
 
         private ILogger Logger { get; }
 
-        public string AnalysisTypeName => "Dependency Analysis";
+        public string Id => "UA101";
```

Comment:
We may want to document rule scheme, but ok for now to get this through

Reply:
I will add some documentation on rule scheme